cmds/cluster/nodestats/main.go

```diff
@@ -66,6 +66,7 @@ func node() *health.Stat {
 	var Stderr [65536]byte
 	n, err := r.Read(Stderr[:])
 	if err != nil && err != io.EOF {
+		//nolint:errorlint
 		errs = errors.Join(errs, fmt.Errorf("stderr read %d bytes, got %v but not %v or nil", n, err, io.EOF))
```

Comment:
is there some way to fix line 70 to get rid of this no lint: no lint:  are creators of technical debt.

Reply:
yeah you right, will update